library/src/main/java/io/nlopez/clusterer/Clusterer.java

```diff
@@ -482,4 +494,11 @@ public void setMarkerAnimation(MarkerAnimation markerAnimation) {
         void onCameraChange(CameraPosition position);
     }
 
+    public interface Listener<T extends Clusterable> {
```

Comment:
The naming of the interface is too generic. Maybe something like OnClusterableClickedListener or ClusterableActionListener would be better.